modules/lges/components/LgeListTabs.tsx

```diff
@@ -0,0 +1,23 @@
+import { HStack, TabList, Tabs } from '@chakra-ui/react';
+import BeetsTab from '~/components/tabs/BeetsTab';
+import { useLgeList } from '~/modules/lges/useLgeList';
+
+export function LgeListTabs() {
+    const { setStatus } = useLgeList();
+    const TABS = [{ id: 'active-upcoming' }, { id: 'ended' }];
```

Comment:
can you define this outside of the component? It will be redefined every render. I know it's a small thing but they add up to a much larger performance impact in a big project

Reply:
Referring to `TABS`

Reply:
Fixed this!